src/main/java/com/gmail/goosius/siegewar/utils/SiegeWarBattleSessionUtil.java

```diff
@@ -297,10 +315,9 @@ private static void sendBattleSessionEndedMessage(Map<Siege, Integer> battleResu
 				lines.add(resultLine);
 			}
 		}
-		//Send message
-		Messaging.sendGlobalMessage(header, lines);
+		return lines;
 	}
-												
+
```

Comment:
unneeded diff

Reply:
i think this was some whitespace that idea removed